lib/utils/index.js

```diff
@@ -161,6 +164,46 @@ async function series (promises) {
 	return results;
 }
 
+/**
+ * Checks to see if the file/directory exists
+ */
+async function exists (path) {
+	let exists = false;
+	try {
+		await fs.stat(path);
+		exists = true;
+	} catch (err) {
+		// lstat throws an error if the directory doesn't exist. 
+		// We don't care about that error because we don't want that
+		// directory to exist. 
+	}
+
+	return exists;
+}
+
+const inflate = promisify(zlib.inflate);
+const defalate = promisify(zlib.deflate);
+
+/**
+ * Decompresses a zlib compressed string.
+ * 
+ * @param text - String to decompress.
+ * @returns - Decompressed string.
+ */
+async function decompress (buffer) {
+	return (await inflate(buffer));
+}
+
+/**
+ * Compresses a string.
+ * 
+ * @param text - String to compress.
+ * @returns - Compressed string.
+ */
+async function compress (text) {
+	return (await defalate(Buffer.from(text), { level: 6 }));
```

Comment:
Do we need `Buffer.from(text)` here? It seems that [deflate works also with strings, not only with buffers](https://nodejs.org/api/zlib.html#zlibdeflatebuffer-options-callback)